concepts/conditionals-if/introduction.md

```diff
@@ -39,7 +39,7 @@ if v := 2 * num; v > 10 {
 } else {
     fmt.Println(num)
 }
-// Output: 14
+// => 14
```

Comment:
Since this is an instance where print functions are used, we should keep the `Output:` wording.